Automattic/wp-calypso
# Offer Reset: add JPCOM master bar to the Pricing page

#### Changes proposed in this Pull Request

* Create a replicate of the jetpack.com master bar and include it in the new Pricing page.
* My plan is to get this merged soon and then in a follow-up PR make all the adjustments left: font size, font family, color, spaces, etc.

#### Caveats

* Nothing has been done regarding fonts so don't pay too much attention to it. This will be fixed in another PR.
* Since we are trying to replicate the behavior and look as much as possible, CSS properties may not be used as we are used to in Calypso (font-sizes, paddings, margins, etc.)
* Neither the language control nor the search inputs are being added in this iteration.

#### Testing instructions

* Run this PR with `yarn start-jetpack-cloud`.
* Visit `http://jetpack.cloud.localhost:3000/pricing`.
* Verify that the new master bar looks like the one in jetpack.com/pricing. 
* Pay attention to the animations, links, colors, and any other behavior you can think of. 
* Don't forget to test it responsiveness.

Fixes 1169247016322522-as-1189368424916239

#### Demo

![image](https://user-images.githubusercontent.com/3418513/92025333-c4dba500-ed35-11ea-8373-15aee56d3ba6.png)
![image](https://user-images.githubusercontent.com/3418513/92025373-d2912a80-ed35-11ea-8be6-4522599ede9e.png)
![image](https://user-images.githubusercontent.com/3418513/92025413-dde45600-ed35-11ea-8db7-efdbc1e7a548.png)
![Kapture 2020-09-02 at 16 08 15](https://user-images.githubusercontent.com/3418513/92026182-028cfd80-ed37-11ea-8b44-c5ee2dc11a26.gif)
